gateway-js/src/typings/graphql.ts

```diff
@@ -0,0 +1,31 @@
+import { GraphQLResolveInfo } from 'graphql';
+
+type GraphQLReferenceResolver<TContext> = (
```

Comment:
This file isn't imported from anywhere? Is it referenced in some config file that I'm missing?  Why is it in gateway-js rather than subgraph-js?

Reply:
oh, i see, there's a second copy in subgraph-js.  well, maybe my other suggestion will let us remove the duplication.  and i guess as long as a file is included in the tsconfig'd project, it doesn't have to be imported for its `declare module` to work?

Reply:
i do wonder though — if the file's not imported from anywhere, will the declare modules work across package? ie, this file generates a .d.ts that isn't sourced from index.d.ts, will it 'take effect' in the same way that it takes effect when you're actively building the package itself?

i mean maybe it doesn't matter since this is kinda an internal implementation detail of `@apollo/subgraph` anyway.